java/ql/src/semmle/code/java/frameworks/javaee/ejb/EJB.qll

```diff
@@ -199,7 +199,7 @@ abstract class EjbInterfaceAnnotation extends Annotation {
     // within the "value" element of this annotation.
     // Uses `getAChildExpr*()` since the "value" element can have type `Class` or `Class[]`.
     exists(TypeLiteral tl | tl = getValue("value").getAChildExpr*() |
-      exists(TypeAccess ta | ta = tl.getTypeName() | result = ta.getType())
+      result = tl.getReferencedType()
```

Comment:
The old code restricted `getTypeName()` to `TypeAccess` even though its result type is only `Expr`. So in theory this could be a behavior change, but I don't think it actually changes the behavior.